src/test/java/org/stellar/sdk/responses/SubmitTransactionResponseTest.java

```diff
@@ -36,6 +36,34 @@ public void testDeserializeTransactionFailureResponse() throws IOException {
     assertEquals(submitTransactionResponse.getExtras().getResultCodes().getTransactionResultCode(), "tx_no_source_account");
   }
 
+  @Test
+  public void testDeserializeFeeBumpTransactionFailureResponse() throws IOException {
+    String json = "{\n" +
```

Comment:
JSON is just made up.

Reply:
great coverage, if possible can you add a second test case to assert no failure in marshaling, if `inner_transaction` is absent in the json response? I'm not 100% sure if that is a required attribute.

Reply:
ah, nvm, I see this [existing test in here](https://github.com/stellar/java-stellar-sdk/pull/466/files#diff-6936d9319b47667cf10508259d1d8c3347d97b891be00ae4722f06f435e1c071R15) which exercised absence of `inner_transaction`

Reply:
Thanks, @sreuland! I mark this as resolved then.